napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -50,6 +52,15 @@
 from napari_plugin_manager.qt_widgets import ClickableLabel
 from napari_plugin_manager.utils import is_conda_package
 
+try:
+    from napari.plugins.utils import PluginStatus
+except ImportError:
+
+    class PluginStatus(Enum):
+        BUSY = auto()
+        IDLE = auto()
```

Comment:
Would this redefinition of [this Enum](https://github.com/napari/napari/pull/6993/files#diff-14f1d686e1b30a48698a5bb7ffde42d49f0860c01f45378849de711b42c15967R24) satisfy equality? Not sure how the magic `auto()` thingy works and wondering if somehow you end up with different placeholder values that fail `==`. If the underlying object is an int I think we are ok, though.